src/Illuminate/Support/Collection.php

```diff
@@ -1116,6 +1116,21 @@ public function take($limit)
     }
 
     /**
+     * Drop the first {$limit} items from the collection.
+     *
+     * @param  int  $limit
+     * @return  static
+     */
+    public function drop($limit)
+    {
+        if ($limit < 0) {
+            $limit = abs($limit);
```

Comment:
I was mostly trying to mimic the behavior of `take`. What do you think the behavior for a negative limit should be?